snaps/features/custom-ui/index.md

```diff
@@ -182,6 +182,55 @@ await snap.request({
 });
 ```
 
+### `Banner`
```

Comment:
Can you move this section up before the `Bold` component?

Reply:
Will do!